.github/workflows/issue-labeled.yml

```diff
@@ -20,14 +20,23 @@ jobs:
           labels: "pending triage, need reproduction"
 
       - name: remove pending
-        if: github.event.label.name == 'enhancement' || github.event.label.name == 'bug' || (contains(github.event.label.name, 'pending triage') == false && startsWith(github.event.label.name, 'bug:') == true)
+        if: startsWith(github.event.label.name, 'p:') == true && contains(github.event.issue.labels.*.name, 'pending triage') == true
```

Comment:
Should we instead list all 5 priorities so we can keep labels like `p1:chore` `p2:nice-to-have` etc?

Reply:
How about `contains(github.event.label.description, '(priority)')`? We could keep label names. But will need to add `(priority)` to every priority labels (we already had them).

Reply:
Sounds good 👍🏼